.github/workflows/push.yml

```diff
@@ -171,10 +172,16 @@ jobs:
           fi
           echo "Using SHA='$SHA' as docker tag sha"
           export DOCKER_TAG=${GITHUB_RUN_NUMBER}_${DOCKER_BRANCH_TAG}_${SHA:0:7}
+          export POLICY_TAG=${DOCKER_BRANCH_TAG}_${GITHUB_RUN_NUMBER}_${SHA:0:7}
```

Comment:
Any idea how hard it would be two make this the new DOCKER_TAG instead of adding this additional one? Are there many places referencing the current format of the dockertag? @Snojo @pshomov

Reply:
I do agree. I did mention that to Petar but he was hesitant to take that step. I think it would require quite a big refactoring session to do.

Reply:
Okay I understand. As I think about it, this would be a necessary step if we would want to do that refactor.

We would need to do something like:
1. Tag the images with both versions of the tag. (This PR)
2. Refactor all references to use the new version of the tag.
3. Remove the old version of the tag.